pkg/resource/cluster.go

```diff
@@ -64,7 +75,35 @@ func (cluster Cluster) Unwrap() *api.CrdbCluster {
 func (cluster Cluster) SetTrue(ctype api.ClusterConditionType) {
 	condition.SetTrue(ctype, &cluster.cr.Status, cluster.InitTime())
 }
+func (cluster Cluster) True(ctype api.ClusterConditionType) bool {
```

Comment:
This needs docs, `True` is an extremely vague name for a method.

Reply:
done